src/include/Context.h

```diff
@@ -463,6 +464,19 @@ class FunctionContext : public Context {
   boost::function<void(int)> m_callback;
 };
 
+class LambdaContext : public Context {
```

Comment:
Sure, no problem. 
Is the plan to substitute all occurrences of boost::function by C++11 native? I can do that afterwards :-)